.circleci/config.yml

```diff
@@ -401,6 +401,9 @@ workflows:
       - acceptanceTests:
           requires:
             - assemble
+      - acceptanceTestsNonMainnet:
+          requires:
+            - assemble
```

Comment:
are we putting this back in

Reply:
no, was just a temporary test to prove it passes before merging, reverting now